schema-engine/datamodel-renderer/src/datamodel.rs

```diff
@@ -136,56 +193,40 @@ mod tests {
         field.default(dv);
 
         model.push_field(field);
-        data_model.push_model(model);
+        data_model.push_model(file_name.to_string(), model);
 
         let mut traffic_light = Enum::new("TrafficLight");
 
         traffic_light.push_variant("Red");
         traffic_light.push_variant("Yellow");
         traffic_light.push_variant("Green");
 
-        data_model.push_enum(traffic_light);
+        data_model.push_enum(file_name.to_string(), traffic_light);
 
         let mut cat = Enum::new("Cat");
         cat.push_variant("Asleep");
         cat.push_variant("Hungry");
 
-        data_model.push_enum(cat);
+        data_model.push_enum(file_name.to_string(), cat);
 
         let mut view = View::new("Meow");
         let mut field = Field::new("id", "Int");
         field.id(IdFieldDefinition::default());
 
         view.push_field(field);
 
-        data_model.push_view(view);
+        data_model.push_view(file_name.to_string(), view);
 
         let expected = expect![[r#"
-            type Address {
-              street String
-            }
-
-            model User {
-              id Int @id @default(autoincrement())
-            }
-
-            view Meow {
-              id Int @id
-            }
-
-            enum TrafficLight {
-              Red
-              Yellow
-              Green
-            }
-
-            enum Cat {
-              Asleep
-              Hungry
-            }
+            [
+                (
+                    "schema.prisma",
+                    "type Address {\n  street String\n}\n\nmodel User {\n  id Int @id @default(autoincrement())\n}\n\nview Meow {\n  id Int @id\n}\n\nenum TrafficLight {\n  Red\n  Yellow\n  Green\n}\n\nenum Cat {\n  Asleep\n  Hungry\n}\n",
+                ),
+            ]
         "#]];
```

Comment:
TODO: join all datamodels like we do in integration tests to improve readability.